app/models/package.rb

```diff
@@ -200,6 +199,13 @@ def create_or_update_location_for_dispatch_from_stockit(dispatched_location, ord
     end
   end
 
+  def deduct_qty_and_make_copies(qty_to_split)
+    1..qty_to_split.times do
```

Comment:
Is it worth wrapping this whole method in an Active Record transaction? If we split the package but something goes wrong along the way, we want to ensure we are in a consistent state (i.e. creating more items but haven't deducted)